src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackageValidation.targets

```diff
@@ -0,0 +1,45 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
```

Comment:
Should we name these `Compatibility.targets`? I would expect to share this with `API Compat` task when I add it?

Reply:
i think we will need to have the same name as the package name to automatically port the package files. we can change the package name to Microrost.DotNet.Compatibility ?